libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -128,6 +131,7 @@ class AC_PrecLand
     AP_Float                    _accel_noise;       // accelometer process noise
     AP_Vector3f                 _cam_offset;        // Position of the camera relative to the CG
 
+    LANDING_TARGET_TYPE         _target_type;       // target type (LANDING_TARGET_TYPE enum)
```

Comment:
Is there a reason why this needs to be saved instead of the method directly asking the backend when the frontend method is called?

Reply:
Connected frontend target_type() to backend target_type()